code/game/machinery/cryopod.dm

```diff
@@ -325,6 +325,12 @@
 		// them win or lose based on cryo is silly so we remove the objective.
 		if(istype(O,/datum/objective/mutiny) && O.target == occupant.mind)
 			qdel(O)
+		else if(is_sacrifice_target(occupant.mind))
+			var/datum/game_mode/cult/cult_mode = ticker.mode
+			O.owner.objectives -= O
+			qdel(O)
+			to_chat(O.owner.current, "<span class='cultitalic'>Our gods chosen gift has gone passed our reach, we have a new goal..Objectives Updated!</span>")
```

Comment:
Well, there must be something that's used to send a message to all the cultists, right? Use that.